apps/ledger-live-desktop/tests/utils/speculos.ts

```diff
@@ -36,6 +40,10 @@ export type Device = {
   appPath: string;
 };
 
+export function setExchangeDependencies(dependencies: Dependency[]) {
```

Comment:
does it make sense to commit code for exchange app if we are not even sure if we will be able to automate it?

Reply:
that was a question i asked during the daily, it seemed to also make sense for other scenarios
I can drop the PR, I thought code would be needed after [this ticket](https://ledgerhq.atlassian.net/browse/B2CQA-2482) is done

Reply:
I would prefer not to commit code into develop which is not in use to be honest, maybe let's keep it in the branch somewhere and get back to it once we are unblocked